pkg/awsutils/awsutils.go

```diff
@@ -1452,7 +1453,7 @@ func (cache *EC2InstanceMetadataCache) tagENIcreateTS(eniID string, maxBackoffDe
 		Tags: tags,
 	}
 
-	_ = retry.RetryNWithBackoff(retry.NewSimpleBackoff(500*time.Millisecond, maxBackoffDelay, 0.3, 2), 5, func() error {
+	_ = retry.NWithBackoff(retry.NewSimpleBackoff(500*time.Millisecond, maxBackoffDelay, 0.3, 2), 5, func() error {
```

Comment:
There seems to be few warnings by code coverage. Is this expected?

Reply:
Yes, the `codecov/patch` function is currently broken. It's supposed to check that new lines of code are covered by tests, but it doesn't handle renames. The full `codecov/project` does work, showing: 54.31% (+0.08%) compared to 0d3daa5